protocols/mdns/CHANGELOG.md

```diff
@@ -22,6 +31,7 @@
 
 - Update to `if-watch` `v2.0.0` and thus the `async` method `Mdns::new` and `TokioMdns::new` becomes synchronous. See [PR 2978].
 
+[PR 3096]: https://github.com/libp2p/rust-libp2p/pull/3096
```

Comment:
This link is in the wrong section, should be under `# 0.42.0`.

Reply:
Right, thanks Thomas, moved it.